### PR TITLE
Update UML diagrams to include Skill and Interview Score

### LINKED ISSUE #115: Update UML diagrams to include Skill and Interview Score

### PR DESCRIPTION
**Pull request by SQ77**
closes #115

### CODE REVIEW

**Review by LemonDrew — APPROVED** [review]
LGTM!

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 1]
```diff
@@ -6,17 +6,21 @@ skinparam classBackgroundColor MODEL_COLOR
...
+AddressBook *-down-> "1" UniqueSkillList
 UniqueTagList -[hidden]down- UniquePersonList
 UniqueTagList -[hidden]down- UniquePersonList
 
 UniqueTagList -right-> "*" Tag
 UniquePersonList -right-> Person
+UniqueSkillList -right-> "*" Skill
 
 Person -up-> "*" Tag
 Person -up-> "1" Tag : Job Status
```
Comment (bigismols) [inline docs/diagrams/BetterModelClassDiagram.puml:18]: Is there a way to make the two arrows pointing to Tag from Person more distinct?

**Review by bigismols — COMMENTED** [review]
Requested a minor change, otherwise LGTM!

File: docs/diagrams/BetterModelClassDiagram.puml (hunk 1 above)
Comment (SQ77) [reply docs/diagrams/BetterModelClassDiagram.puml]: Ok, they should be more distinct now

**Pull request merged**